verticapy/toolbox.py

```diff
@@ -389,7 +389,6 @@ def default_model_parameters(model_type: str):
             "max_ntree": 10,
             "max_depth": 5,
             "nbins": 32,
-            "objective": "squarederror",
```

Comment:
Why?

Reply:
Because of versioning issues. It created numerous bugs. Now squarederror is for Regressor and crossentropy for classifiers. When we will get more options, I'll add it again.